client/src/utils/testFunctions.ts

```diff
@@ -0,0 +1,64 @@
+import { colMaps } from '@/utils/colormap';
```

Comment:
are these test functions? I'd argue 'normaliseMagnitude' and 'generateSampleRecording' could be used for multiple purposes

Reply:
they are used only for test functions. They may be used for something else later - no way to know.

Reply:
I agree about normaliseMagnitude ( but this puts in question the whole validity of the tests ( I mean having some fixed values to tests with that function would make sense )

Reply:
I think the best option to test this it is to use well known values for normalisation instead of a function.

Reply:
I prefer Jade's solution. It enables more tests.